modules/installation-complete-user-infra.adoc

```diff
@@ -5,6 +5,13 @@
 // * installing/installing_restricted_networks/installing-restricted-networks-vsphere.adoc
 // * installing/installing_vsphere/installing-vsphere.adoc
 
+ifeval::["{context}" == "installing-restricted-networks-vsphere"]
```

Comment:
I know AWS uses a different file, but just wanted to double check that the "register your cluster" step didn't need to be added there too.